corehq/apps/app_manager/templates/app_manager/load_save_questions.html

```diff
@@ -0,0 +1,27 @@
+<ul class="fa-ul" data-bind="foreach: {{ questions }}"><!-- Each of these is an object containing a condition and a question -->
```

Comment:
Can this go in `partials`? App manager templates aren't all that organized, but that'd help a little.